### PR TITLE
Jared dev

### PR DESCRIPTION
**Pull request by jared-cain**
Create basic Dockerfile for compute worker and wire up to docker-compose.yml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ To see Codalab Competition's in action, visit [www.competitions.codalab.org](htt
 The CodaLab community forum is hosted on Google Groups.
 - [CodaLabDev Google Groups Forum](https://groups.google.com/forum/#!forum/codalabdev)
 
+## Tests
+
+To run tests:
+
+```python manage.py test```
+
```
Comment (ckcollab) [inline README.md]: Is this PR against the right branch and such?

File: codalab/codalabtools/compute/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+anyjson==0.3.3
+argparse
+azure==0.7.1
```
Comment (ckcollab) [inline codalab/codalabtools/compute/requirements.txt]: What's the purpose of the extra requirements?

File: docker-compose.yml [hunk 3]
```diff
@@ -7,3 +7,10 @@ services:
       - "MYSQL_DATABASE=codalab_website"
     ports:
         - 3306:3306
+  worker:
```
Comment (ckcollab) [inline docker-compose.yml]: Will you add instructions to the readme for how to run this all together?

File: codalab/codalabtools/compute/requirements.txt (hunk 2 above)
Comment (jared-cain) [inline codalab/codalabtools/compute/requirements.txt]: just noticed that myself. fixing now

File: README.md (hunk 1 above)
Comment (ckcollab) [inline README.md]: I think it is, maybe I just never pushed this to develop... Thanks! :)

File: README.md [hunk 4]
```diff
@@ -15,6 +15,19 @@ To see Codalab Competition's in action, visit [www.competitions.codalab.org](htt
 The CodaLab community forum is hosted on Google Groups.
 - [CodaLabDev Google Groups Forum](https://groups.google.com/forum/#!forum/codalabdev)
 
+## Compute worker
+
+To start the compute worker under project directory:
+
+`docker-compose up worker`
```
Comment (ckcollab) [inline README.md]: How long would it take to modify this to run everything -- django, compute worker, site worker, celery, and mysql? Could we run compute worker independently still with that kind of setup?

Comment (jared-cain) [inline README.md]: Im not sure how long it would take, but definitely can be done.

File: codalab/codalabtools/compute/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+RUN mkdir -p /app
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
+RUN wget -O codalabtools.py https://raw.githubusercontent.com/jared-cain/codalab-competitions/jared-dev/codalab/codalabtools/__init__.py && \
```
Comment (ckcollab) [inline codalab/codalabtools/compute/Dockerfile]: Can we not hard code this? Should be able to grab relative to our directory, somehow?

File: codalab/codalabtools/compute/worker.py [hunk 6]
```diff
@@ -27,11 +27,11 @@
 from zipfile import ZipFile
 
 # Add codalabtools to the module search path
-sys.path.append(dirname(dirname(dirname(abspath(__file__)))))
+#sys.path.append(dirname(dirname(dirname(abspath(__file__)))))
```
Comment (ckcollab) [inline codalab/codalabtools/compute/worker.py]: Was this necessary to comment out or just testing stuff?

Comment (jared-cain) [inline codalab/codalabtools/compute/worker.py]: just testing stuff. it can definitely go back

File: codalab/codalabtools/compute/Dockerfile (hunk 5 above)
Comment (jared-cain) [inline codalab/codalabtools/compute/Dockerfile]: yeah ive been trying to make it work with the volumes key in docker compose but was unsuccessful. let me take another shot at it

File: docker-compose.yml (hunk 3 above)
Comment (jared-cain) [inline docker-compose.yml]: did you need more instructions on this?

File: codalab/codalabtools/compute/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+RUN apt-get install -y build-essential && \
+    apt-get install -y psmisc && \
+    pip install -r requirements.txt && \
+    mkdir /tmp/codalab && \
+    chmod -R ugo+rwx /tmp/codalab
+    
+
+#RUN mkdir codalabtools
+#RUN wget -O codalabtools.py https://raw.githubusercontent.com/jared-cain/codalab-competitions/jared-dev/codalab/codalabtools/__init__.py && \
+#    wget https://raw.githubusercontent.com/jared-cain/codalab-competitions/jared-dev/codalab/codalabtools/azure_extensions.py
```
Comment (ckcollab) [inline codalab/codalabtools/compute/Dockerfile]: This can be removed I believe?

File: docker-compose.yml (hunk 3 above)
Comment (ckcollab) [inline docker-compose.yml]: No sir that's good so far!

File: codalab/codalabtools/compute/Dockerfile [hunk 8]
```diff
@@ -0,0 +1,20 @@
+FROM continuumio/anaconda:4.3.0
+RUN mkdir -p /app
+COPY . /app
+WORKDIR /app/codalab/codalabtools/compute
+RUN apt-get install -y build-essential && \
+    apt-get install -y psmisc && \
```
Comment (ckcollab) [inline codalab/codalabtools/compute/Dockerfile]: Do these need to be chained? Can we have separate `RUN`s?

**Pull request closed without merging**